javascript/javascript2/week3/hyf-homework-week3/exercise.js

```diff
@@ -0,0 +1,173 @@
+console.log('Js 2 homework week 3')
+       
+// step 1
+setTimeout(() => {
+    console.log("Called after 2.5 seconds")
+}, 2500);
+
+    // OR
+// window.addEventListener('load',alreadyLoaded)
+// function alreadyLoaded() {
+//     setTimeout(() => {
+//         console.log("Called after 2.5 seconds")
+//     }, 2500);  
+// }
+
+//         // step 2
+
+function loadString(delay,stringToLog) {
+    setTimeout(() => {
+        console.log(stringToLog)
+    }, delay);
+}
+loadString(5000,"Called after 5 seconds");
+
+//         // step 3
+
+const firstButton = document.querySelector('#btn1')
+firstButton.addEventListener('click',() =>{
+    loadString(5000,"Button clicked after 5 seconds");
+})
+
+        // step 
+
+const forEarth = (() => {
+    console.log('Earth')
+})
+// forEarth()
+
+const forSaturn= (() => {
+    console.log('Saturn')
+})
+// forSaturn()
+
+ function planetLogFunction(callBack){
+    return callBack()
+}
+planetLogFunction(forEarth)
+function planetLogFunction(callBack){
+    return callBack()
+}
```

Comment:
You don't need to define the same function 2 times here. Just calling that function twice with different parameters is how you do